ui/views.py

```diff
@@ -245,16 +245,17 @@ def extra_context(self):
                 )
             qs_prefix = "?{0}&".format("&".join(qs_prefix))
 
-        context.update({
-            "repo": self.repo,
-            "perms_on_cur_repo": get_perms(self.request.user, self.repo),
-            "vocabularies": {
-                k: v
-                for k, v in context["facets"]["fields"].items()
-                if k in vocabularies
-            },
-            "qs_prefix": qs_prefix,
-        })
+        if "fields" in context["facets"]:
```

Comment:
Note: Nothing changed here except for this `if` statement, and indenting the `context.update` block.